docs/docs/gatsby-on-linux.md

```diff
@@ -64,10 +62,11 @@ Or to install all at the same time and approve `(y)` all installs:
 sudo apt update && sudo apt -y upgrade && sudo apt install build-essential && sudo apt install git && sudo apt install libpng-dev
```

Comment:
These commands are specific to Debian distros, it would be great if you provide similar docs for other distros like cent, arch, etc.

Reply:
I have never used them but I will look them up.

Reply:
Marking as resolved since @laurieontech fixed this.